ports/xeus/portfile.cmake

```
@@ -1,11 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO QuantStack/xeus
-    REF 424b7cd177886906a59eee535b7de59088461910 # 0.24.1
-    SHA512 877ca45bf649b567a9921d3e8f0adb0299dbe956978bd6e217d0c06617cf3466d08d90d607fd33e129089472e1a96ecec78b1fc21346bc13ba268168a5a6b068
+    REF 0f6327a2782181e7ded9729abb32b7d8eb690aea # 0.24.3
+    SHA512 2c0ccd1bebf487a9a73e73ecfb74b7605756652b2a84c71e739d7b2d8923960594c025e36d75cec850c5f0e38614a20299feccea6cfbe9ea0f66bdf315af02b4
     HEAD_REF master
     PATCHES
-		Fix-Compile-nlohmann-json.patch
+        Fix-Compile-nlohmann-json.patch
 )
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" BUILD_STATIC_LIBS)

```

```suggestion
string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" BUILD_SHARED_LIBS)
```
Dynamic is still required to be added here.

---

The OPTIONS should be like this:

```
-DXEUS_BUILD_STATIC_LIBS=${BUILD_STATIC_LIBS}
-DXEUS_BUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
```